### DISCUSSION TITLE
Scripting command "Substring" seems unavailable in Tasmota

### DISCUSSION THREAD

**Discussion by ygelt** [arendst/Tasmota#20007] [Q&A] [answered]
Hi, after a lot of search I am still unable to find a substring command in Tasmota ... does it exist ?

**Comment by sfromis** [comment, marked as answer]
Read the docs. https://tasmota.github.io/docs/Scripting-Language/
The word "substring" occurs just one place in that page.....

**Reply by barbudor** [reply]
> sb(svar p n) = gets a substring from svar at position p (if p<0 counts from end) and length n

**Reply by ygelt** [reply]
Many thanks, I had issue testing it so I thought these functions were not implemented on ESP8266, I was wrong.
